ditto/utils/webhooks.py

```diff
@@ -9,23 +9,27 @@
 
 class EmbedWebhookLogger:
     _to_log: list[discord.Embed]
+    _files_to_log: list[discord.File]
 
     def __init__(self, webhool_url: str) -> None:
         self.loop = asyncio.get_event_loop()
         self._webhook_url = webhool_url
         self._to_log = []
+        self._files_to_log=[]
 
         self._session = aiohttp.ClientSession()
         self._webhook = discord.Webhook.from_url(self._webhook_url, session=self._session)
 
         # setup loop
         self._loop.start()
 
-    def log(self, embed: discord.Embed) -> None:
+    def log(self, embed: discord.Embed,files: list[discord.File]=[]) -> None:
         self._to_log.append(embed)
+        self._files_to_log.extend(files)
 
     @tasks.loop(seconds=5)
     async def _loop(self) -> None:
         while self._to_log:
             embeds = [self._to_log.pop(0) for _ in range(min(10, len(self._to_log)))]
-            await self._webhook.send(embeds=embeds)
+            files=[self._files_to_log.pop(0) for _ in range(min(10,len(self._files_to_log)))]
```

Comment:
Because of how messages work chunking files like how embeds was done probably isn't what we want.

For example, say that we've got 3 embeds waiting to go and the 2nd and 3rd messages have associated files.
Our webhook message would end up looking like

- Embed1
- Embed2
- Embed3
- Embed2File
- Embed3File

Essentially impossible to associate them.
What might make more sense 
Is go store a map of embed "IDs" (I guess the message ID, though could just be like hash(embed)) to their associated files.

And have the chunking logic stop chunking the moment it gets an embed with associated files, therefore the last embed in each webhook message is guaranteed to have it'd associated files right under it.